contrib/forms/handwritten-notes/new.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Create new handwritten notes area
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Tyler Wrenn <tyler@tylerwrenn.com>
+ * @copyright Copyright (c) 2020 Tyler Wrenn <tyler@tylerwrenn.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ **/
+
+require_once("../../globals.php");
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+
+if (!$encounter) {
+    die(xlt("Internal Error: We do not seem to be in an encounter!"));
+}
+
+if ($_POST) {
+    if (!empty($_POST['docid'])) {
+        $setdocid = "UPDATE `form_handwritten` SET `value` = " . $_POST['docid'] . " WHERE `name` = 'doc_category'";
+        sqlStatement($setdocid);
```

Comment:
need to bind to prevent sql injection:
```
$setdocid = "UPDATE `form_handwritten` SET `value` = ? WHERE `name` = 'doc_category'";
sqlStatement($setdocid, [$_POST['docid']]);
```